.travis.yml

```diff
@@ -30,12 +30,12 @@ before_install:
 install:
     - sh travis.sh bootstrap
     - sh travis.sh install_package_control
-    - pip3 install python-coveralls pycodestyle;
+    - pip3 install python-coveralls flake8;
 
 script:
     - sh travis.sh run_tests --coverage
     - sh travis.sh run_syntax_tests
-    - pycodestyle --statistics --count
+    - flake8 cmdbox
```

Comment:
What should `cmdbox` be? If I checkout this PR and run `flake8 .` I still get errors.

Reply:
Not much considerration, just copy pasted the first I have over https://simpleisbetterthancomplex.com/packages/2016/08/05/flake8.html

falake8 . gives me error too. I will ammend the commit to run flake8 . . Then fix those errors too